lua/neo-tree/setup/netrw.lua

```diff
@@ -5,14 +5,15 @@ local command = require("neo-tree.command")
 local M = {}
 
 local get_position = function(source_name)
-  local pos = utils.get_value(M, "config." .. source_name .. ".window.position", "left")
+  local nt = require("neo-tree")
+  local pos = utils.get_value(nt.config, source_name .. ".window.position", default_pos, true)
```

Comment:
It doesn't actually have any effect with how this is used right now, but I think the `default_pos` variable is undefined here and should be replaced with `"left"`.

Reply:
Right, fixed that now.